src/components/activities/patientDetailsActivity/tabsConfig.tsx

```diff
@@ -4,13 +4,48 @@ import SkeletonLoader from "../../accessories/skeletonLoader/SkeletonLoader";
 import PatientOPD from "../../accessories/patientOPD/patientOPD";
 import PatientTriage from "../../accessories/patientTriage/PatientTriage";
 import PatientSummary from "../../accessories/patientSummary/PatientSummary";
-import PatientDetailsContent from '../patientDetailsActivityContent/PatientDetailsActivityContent';
+import PatientDetailsContent from "../patientDetailsActivityContent/PatientDetailsActivityContent";
+import { withPermission } from "../../../libraries/permissionUtils/withPermission";
 
 export const patientDetailTabs: TTabConfig = [
-  { label: "Summary", path:"/summary", content: <PatientDetailsContent title="Summary" content={<PatientSummary />} /> },
-  { label: "OPD", path:"/OPD", content: <PatientDetailsContent title="OPD" content={<PatientOPD />} /> },
-  { label: "Triage", path:"/triage", content: <PatientDetailsContent title="Triage" content={<PatientTriage />} /> },
-  { label: "Therapy", path:"/therapy", content: <PatientDetailsContent title="Therapy" content={<SkeletonLoader />} /> },
-  { label: "Booking", path:"/booking", content: <PatientDetailsContent title="Booking" content={<SkeletonLoader />} /> },
+  {
+    label: "Summary",
+    path: "/summary",
+    content: (
+      <PatientDetailsContent
+        title="Summary"
+        content={withPermission("summary.read")(PatientSummary)}
```

Comment:
@almeida1492 I tried to pull up this check, but going further would mean:
- to change TTabConfig signature (it's ok for me)
- to add a loader indicator in permission provider, because permissions are empty while loading, and current logic of this component would redirect to default route